queries.sql

```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon';
+
+SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+
+SELECT name from animals WHERE neutered = true AND escape_attempts < 3;
+
+SELECT date_of_birth from animals WHERE name IN ('Agumon', 'Pikachu');
+
+SELECT name, escape_attempts FROM animals WHERE weight_kg < 10.5;
```

Comment:
- [ ] It would be better to get accurate result when the `weight_kg > 10.5` instead of `weight_kg < 10.5` as we can see in the project requirements below
![Screenshot from 2023-05-03 18-58-15](https://user-images.githubusercontent.com/46388113/236015297-cf542548-595d-4b21-8fad-12242443dfb2.png)

Reply:
Thank you @addod19 👍  I Fixed the typo.